### PR TITLE
Bug 1329233 - Add docs for CrashSummaryView

### PR DESCRIPTION
**Pull request by maurodoglio**

### CODE REVIEW

File: docs/CrashSummary.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+spark-submit \
+    --master yarn \
+    --deploy-mode client \
+    --class com.mozilla.telemetry.views.CrashSummaryView \
+    target/scala-2.11/telemetry-batch-view-1.1.jar \
+    --outputBucket example_bucket \
+    --from 20160412 \
+    --to 20160428
+```
+Notes:
```
Comment (harterrt) [inline docs/CrashSummary.md]: This is probably worth it's own "Update Frequency" section

File: docs/CrashSummary.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+* This dataset will be updated daily via the [telemetry-airflow](https://github.com/mozilla/telemetry-airflow) infrastructure.
+* The job DAG runs every day shortly after midnight UTC.
+* The job saves the resulting data to S3 as [Parquet](https://parquet.apache.org/)-serialized [DataFrames](https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/DataFrame.html), under prefixes of the form `crash_summary/v1/submission_date=(YYYYMMDD SUBMISSION DATE)/` in the `telemetry-parquet` bucket.
+* The Crash Summary data can be accessed from Spark via clusters launched at [analysis.telemetry.mozilla.org](https://analysis.telemetry.mozilla.org/) or via [Presto](https://prestodb.io/) at `sql.telemetry.mozillla.org`. The table is called `crash_summary`.
+
+Schemas and Making Queries
+--------------------------
+
+The dataset was built with the long term goal of providing a base for CrashAggregates.
```
Comment (harterrt) [inline docs/CrashSummary.md]: This gives a lot of context. You may want to move this up to the introduction (near line 3).

File: docs/CrashSummary.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    --class com.mozilla.telemetry.views.CrashSummaryView \
+    target/scala-2.11/telemetry-batch-view-1.1.jar \
+    --outputBucket example_bucket \
+    --from 20160412 \
+    --to 20160428
+```
+Notes:
+
+* This dataset will be updated daily via the [telemetry-airflow](https://github.com/mozilla/telemetry-airflow) infrastructure.
+* The job DAG runs every day shortly after midnight UTC.
```
Comment (harterrt) [inline docs/CrashSummary.md]: Please remove "DAG"

**Comment by maurodoglio** [comment]
This is ready for another look, sorry it took so long.

**Pull request merged**